src/tests/functional/plugin/cpu/subgraph_tests/src/seq_native_order.cpp

```diff
@@ -265,14 +287,27 @@ const std::vector<size_t> inputSizes = {
     1, 10
 };
 
-const std::vector<InputShapeParams> inShapeParams = {
+const std::vector<InputShapeParams> inShapeParams_dynamic = {
     InputShapeParams{std::vector<ov::Dimension>{-1, -1}, std::vector<TargetShapeParams>{TargetShapeParams{3, 8},
                                                                                         TargetShapeParams{10, 2}}},
-    InputShapeParams{std::vector<ov::Dimension>{{1, 15}, {1, 15}}, std::vector<TargetShapeParams>{TargetShapeParams{3, 8},
-                                                                                                  TargetShapeParams{10, 2}}}
+    InputShapeParams{std::vector<ov::Dimension>{{1, 15}, {1, 5}}, std::vector<TargetShapeParams>{TargetShapeParams{7, 5},
+                                                                                                 TargetShapeParams{10, 2}}},
+    InputShapeParams{std::vector<ov::Dimension>{{1, 8}, 9}, std::vector<TargetShapeParams>{TargetShapeParams{7, 9},
+                                                                                           TargetShapeParams{8, 9}}},
+    InputShapeParams{std::vector<ov::Dimension>{6, {1, 5}}, std::vector<TargetShapeParams>{TargetShapeParams{6, 5},
+                                                                                           TargetShapeParams{6, 2}}},
+};
+
+const std::vector<InputShapeParams> inShapeParams_static = {
+    InputShapeParams{std::vector<ov::Dimension>{10, 2}, std::vector<TargetShapeParams>{TargetShapeParams{10, 2},
+                                                                                       TargetShapeParams{10, 2}}}
 };
 
-std::vector<std::vector<std::string>> activations = {
+std::vector<std::vector<std::string>> activations_gru = {
```

Comment:
Should it be RNN activation here as well?

Reply:
I separate this activations because we don't support any activations other than `{"sigmoid", "tanh"}` for GRU and `{"sigmoid", "tanh", "tanh"}` for LSTM. RNN supports any `activations`. So we no need to create separate activations vector for RNN. Added postfix `_support` for vectors name.